### ISSUE TITLE
kind registry mirrors should redirect from name:port to http://name:port

### ISSUE THREAD

**Issue by cwlbraa** [tilt-dev/ctlptl#49]
When provisioning a kind cluster + a local registry, any attempt to use images via hostFromClusterNetwork causes errors because containerd tries to communicate with the insecure registry via https.

https://github.com/tilt-dev/ctlptl/blob/472eb5bdf7dff68bd99c527e85f7028296b5aca6/pkg/cluster/admin_kind.go#L58-L63

Adding an additional mirror to force containerd to access registry-name:port via http would fix the problem. I confirmed this by docker execing into the kind container, modifying the containerd config.toml, and restarting the containerd service.

My current ctlptl ymls:

cluster.yml
```
---
apiVersion: ctlptl.dev/v1alpha1
kind: Cluster
product: kind
registry: registry.local
```
registry.yml
```
---
apiVersion: ctlptl.dev/v1alpha1
kind: Registry
port: 5000
name: registry.local
```

Proud to grab first issue! This is a slick little tool.

**Comment by nicks** [comment]
Aww thanks @cwlbraa  :blush: 

Ya I like this idea. Out of curiosity: what are you using hostFromClusterNetwork for? I ask some of the use-cases I've heard are WILD.

**Comment by cwlbraa** [comment]
I'm not using it directly, but when trying to run [knative/serving e2e tests](https://github.com/knative/serving/tree/master/test) against a local cluster, I needed to provide a registry url that is accessible both for containerd image pulls AND for controllers to talk to. In the current setup, localhost works for containerd, but the controllers can't reach the registry on localhost.

**Comment by nicks** [comment]
this should be available in ctlptl v0.2.2!
